packages/beeq/src/components/alert/bq-alert.tsx

```diff
@@ -190,6 +211,13 @@ export class BqAlert {
             [`bq-alert bq-alert__${this.type}`]: true,
             'is-sticky': this.sticky,
           }}
+          data-transition-enter="transform transition ease-out duration-300"
+          data-transition-enter-start="translate-y-xs opacity-0 sm:translate-y-0 sm:translate-x-s"
+          data-transition-enter-end="translate-y-0 opacity-100 sm:translate-x-0"
+          data-transition-leave="transform transition ease-in duration-100"
+          data-transition-leave-start="translate-y-0 opacity-100 sm:translate-x-0"
+          data-transition-leave-end="-translate-y-xs opacity-0 sm:translate-y-0 sm:translate-x-s"
```

Comment:
I would rather use an opacity transition for the enter/leave animation. Remember that Alert are intended to appear right were they are placed (except for the sticky version)

```suggestion
          data-transition-enter="transition ease-out duration-300"
          data-transition-enter-start="opacity-0"
          data-transition-enter-end="opacity-100"
          data-transition-leave="transition ease-in duration-200"
          data-transition-leave-start="opacity-100"
          data-transition-leave-end="opacity-0"
```

Reply:
Also, we might need to change the `Watch('open')` method a bit.

```ts
  @Watch('open')
  handleOpenChange() {
    this.autoDismissDebounce?.cancel();

    if (!this.open) {
      this.handleHide();
      return;
    }

    this.handleShow();

    if (this.autoDismiss) {
      this.autoDismissDebounce();
    }
  }
```

With the current method, the `handleShow()` and `handleHide()` methods are not hit when the open property values change, meaning no animation is shown, but the change proposed should fix that.


https://github.com/Endava/BEEQ/assets/328492/769484e9-6b04-4d68-b71e-d6722f17eed3